hack/run-lint.sh

```diff
@@ -28,4 +29,19 @@ then
 fi
 
 # configured by .golangci.yml
-exec env GO111MODULE=on "$gopath/bin/golangci-lint" run
+"$gopath/bin/golangci-lint" run
+
+# install impi that ensures import grouping is done consistently
+if ! [[ -x "${gopath}/bin/impi" ]]
+then
+   echo >&2 'Installing impi'
```

Comment:
This block should be moved into a function as for `install_gox` in `hack/install-gox.sh`.

Reply:
done.